charts/pulsar/templates/pulsar-manager-service.yaml

```diff
@@ -21,7 +21,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: "{{ template "pulsar.fullname" . }}-{{ .Values.pulsar_manager.component }}"
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.pulsar_manager.component }}-frontend"
```

Comment:
Changing this service name might break existing integrations. Can we leave it as is in this PR?

Reply:
@michaeljmarshall Hi, Michael, I fail to understand why this is a breaking change. It's an extension, not a change (apropos open-close-principle). If there is a reason why this would break, what else do you suggest we could submit in the next PR? Any strategy from the team?